components/locale_core/src/langid.rs

```diff
@@ -30,18 +30,6 @@ use alloc::borrow::Cow;
 /// This operation normalizes syntax to be well-formed. No legacy subtag replacements is performed.
 /// For validation and canonicalization, see `LocaleCanonicalizer`.
 ///
```

Comment:
Issue: Please restore the `# Ordering` block up here, above the examples, because it is one of the most common questions we've gotten. We literally get people opening PRs multiple times per year to add the "missing" Ord impls. In fact I might even put it above `# Parsing`. I suggested some concise language in my previous comment.

Reply:
It can't be above examples because now it contains a code block. The ordering example should not be the first example.